packages/mui-lab/src/CalendarPicker/CalendarPicker.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {
+  CalendarPicker as XCalendarPicker,
+  CalendarPickerProps,
+} from '@mui/x-date-pickers/CalendarPicker';
+
+let warnedOnce = false;
+
+const warn = () => {
+  if (!warnedOnce) {
+    console.warn(
+      [
+        'MUI: The CalendarPicker component was moved from `@mui/lab` to `@mui/x-date-pickers`',
+        'The import from `@mui/lab` will be removed in the first release of July 2022.',
```

Comment:
I'm not sure to understand what "import from" means. 

```suggestion
        'The component will no longer be exported from `@mui/lab` in the first release of July 2022.',
```